index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>login page</title>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+    <link rel="stylesheet" href="css/index.css">
+    <link rel="stylesheet" href="css/bootstrap.css">
+</head>
+
+<body>
+    <nav>Crowdstaffing</nav>
+
+             <form class="form-container">
+                 
+                 <div class="card">
+                    <h1> Sign into Crowdstaffing</h1>
+                    <p> Enter your Crowdstaffing email address or username and password  </p>
+                 </div>
+
+                  <div class="card">
+                    <label for="username"><b></b></label>
+                    <input type="text" class="form-control" id="username" aria-describedby="emailHelp" placeholder="Enter Username">
+                    
+                
+                    <label for="exampleInputPassword1"></label>
+                    <input type="password" class="form-control" id="exampleInputPassword1" placeholder="Enter Password">
+                        
+
+                    <button type="submit" class="btn btn-primary">Login</button>
+
+
+                    <div class="form-group form-check">
+                        <input type="checkbox" class="form-check-input" id="exampleCheck1">
+                        <label class="form-check-label" for="exampleCheck1">Remember me</label>
+                      
+                    </div>
+                    <span class="psw"><a href="#">Forgot my password</a></span>
+                  </div>
+                
+  
+                </form>
+
+                
+          
+ <script src="js/login.js" type="text/javascript">  </script>
+ <script src="js/bootstrap.js" type="text/javascript">  </script>
+
+<script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
```

Comment:
@indalgit - every single file should be in separate folder i.e in **js** folder.